app/presenters/instance_presenter.rb

```diff
@@ -26,6 +26,12 @@ def domain_count
     Rails.cache.fetch('distinct_domain_count') { Account.distinct.count(:domain) }
   end
 
+  def open_registrations
```

Comment:
I think I would prefer to leave `open_registrations` as a delegation to Setting here in the InstancePresenter.

I'm not sure it needs to know about `max_users`, unless we're going to display that in a view somewhere.

We can probably skip instance presenter altogether and create a method in the registrations controller like `new_registrations_accepted?` that checks both open_registrations and max_users, and has similar spec coverage to what you added below. If it would be easier to test that via a helper or separate class, that's fine too.